File: JS-Basics/03-Conditional-Statements-Lab/p07_Area_of_Figures.js
function areaOfFigures(input) {
    let figures = input[0];
    let area = 0;
    if (figures == 'square') {
        let a = Number(input[1]);
        area = a * a;

    } else if (figures == 'rectangle') {
        let a = Number(input[1]);
        let b = Number(input[2]);
        area = a * b;
        
    } else if (figures == 'circle') {
        let r = Number(input[1]);
        area = r * r * Math.PI;
        
    } else if (figures == 'triangle') {
        let a = Number(input[1]);
        let h = Number(input[2]);
        area = (a * h) / 2;
    }
    console.log(area.toFixed(3));
}

areaOfFigures(["square", "5"]);
areaOfFigures(["rectangle","7","2.5"]);
areaOfFigures(["circle","6"]);
areaOfFigures(["triangle","4.5","20"]);

File: Programming-Basic/03-Conditional-Statements-Lab/p07_Area_Of_Figures.java
import java.util.Scanner;

public class p07_Area_Of_Figures {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double area = 0.0;

        //      Прочитане на входни данни
        String figure = scanner.nextLine();

//      Пресмятания
        if (figure.equals("square")) {
            double a = Double.parseDouble(scanner.nextLine());
            area = a * a;

        } else if (figure.equals("rectangle")) {
            double a = Double.parseDouble(scanner.nextLine());
            double b = Double.parseDouble(scanner.nextLine());
            area = a * b;

        } else if (figure.equals("circle")) {
            double r = Double.parseDouble(scanner.nextLine());

            area = r * r * Math.PI;

        } else if (figure.equals("triangle")) {
            double a = Double.parseDouble(scanner.nextLine());
            double h = Double.parseDouble(scanner.nextLine());
            area = a * h / 2;

        }
        System.out.printf("%.3f", area);

//      Печатане на резултата sout
    }
}
